### PR TITLE
adds documentation for getting start with the scaffolder

### PR DESCRIPTION
**Pull request by punkle**
ch4365
 
 Preview link: https://deploy-preview-494--roadie.netlify.app/docs/getting-started/scaffolding-components/

### CODE REVIEW

**Comment by punkle** [comment]
How do I create a demo page. Ive seen people linking them.

**Comment by punkle** [comment]
I see the link is in the deploy-preview check. :)

File: content/docs/getting-started/scaffolding-components/index.md [hunk 1]
```diff
@@ -0,0 +1,101 @@
...
+
+<div style="position: relative; padding-bottom: 52.42718446601942%; height: 0;"><iframe src="https://www.loom.com/embed/da6159c4ca39458cb5ad03138612a5a3" frameborder="0" webkitallowfullscreen mozallowfullscreen allowfullscreen style="position: absolute; top: 0; left: 0; width: 100%; height: 100%;"></iframe></div>
+
+### Step One. Enable the scaffolder
+
+Visit the `Administration > Settings > Scaffolder` and make sure the enabled check box is selected, press save and apply and restart.
+
+### Step Two. Create a template.yaml file in a repository
+
+You can either create the following file in your own repository or use our example that is in a public repository. https://github.com/RoadieHQ/sample-service/blob/main/demo_template.yaml
```
Comment (dtuite) [inline content/docs/getting-started/scaffolding-components/index.md:26]:
```suggestion
You can either create the following file in your own repository or [use the public example](https://github.com/RoadieHQ/sample-service/blob/main/demo_template.yaml) we prepared. 
```

File: content/docs/getting-started/scaffolding-components/index.md [hunk 2]
```diff
@@ -0,0 +1,101 @@
...
+
+<details>
+  <summary>Expand here to see an example template.yaml</summary>
```
Comment (dtuite) [inline content/docs/getting-started/scaffolding-components/index.md:29]: Love that this is hidden until required. Very nice touch!

File: content/docs/getting-started/scaffolding-components/index.md [hunk 3]
```diff
@@ -0,0 +1,101 @@
...
+title: Scaffolding Components in Backstage
+lastUpdated: '2021-11-24T14:00:00.0Z'
+description: How to create components such as services or websites using the Backstage scaffolder.
+---
+
+## Introduction
+
+This tutorial will show you how to create a basic scaffolder template in Backstage.
+
+## Prerequisite
```
Comment (dtuite) [inline content/docs/getting-started/scaffolding-components/index.md:11]: You have to enable it in the settings too right? An admin could turn off the scaffolder for a tenant, then another user of the same tenant could read this and be confused.

**Review by dtuite — CHANGES_REQUESTED** [review]

File: content/docs/getting-started/scaffolding-components/index.md (hunk 3 above)
Comment (punkle) [reply content/docs/getting-started/scaffolding-components/index.md:11]: The first step explains how to enable it. Are you suggesting that I move that up to the prerequisites?

Just not sure what you mean.

File: content/docs/getting-started/scaffolding-components/index.md (hunk 2 above)
Comment (punkle) [reply content/docs/getting-started/scaffolding-components/index.md:29]: Thank you

File: content/docs/getting-started/scaffolding-components/index.md (hunk 3 above)
Comment (dtuite) [reply content/docs/getting-started/scaffolding-components/index.md:11]: Ah yes. Indeed it does!

**Review by dtuite — APPROVED** [review]

**Pull request merged**